PowerShellTools/DebugEngine/BreakpointManager.cs

```diff
@@ -145,14 +145,7 @@ public void SetBreakpoint(ScriptBreakpoint breakpoint)
 
             try
             {
-                if (Debugger.DebuggingService.GetRunspaceAvailability() == RunspaceAvailability.Available)
-                {
-                    Debugger.DebuggingService.SetBreakpoint(new PowerShellBreakpoint(breakpoint.File, breakpoint.Line, breakpoint.Column));
-                }
-                else if (Debugger.IsDebuggingCommandReady)
-                {
-                    Debugger.DebuggingService.ExecuteDebuggingCommandOutNull(string.Format(DebugEngineConstants.SetPSBreakpoint, breakpoint.File, breakpoint.Line));
-                }
+                Debugger.DebuggingService.SetBreakpoint(new PowerShellBreakpoint(breakpoint.File, breakpoint.Line, breakpoint.Column), Debugger.IsDebuggingCommandReady);

```

Comment:
Yes. So basically the problem was that in the original version, the ExecuteDebuggingCommandOutNull would use the local file name. My options were to either move all logic into the service (which I ended up doing). Or making a new service method that given a file, would return the corresponding remote file name to you/the passed in file name if there wasn't match. I chose the first since the second would require an extra WCF call, but I definitely think that the second option would be a more understandable refactor, and the new method could be potentially end up being used in other places. What do you think @EricMSFT? Because I can totally re-refactor to the second option.